src/components/Comments/WriteComment.js

```diff
@@ -68,7 +68,7 @@ const WriteComment = ({
     if (isKeyboardOpen) {
       commentInputRef.current.focus();
     }
-  }, [isKeyboardOpen, commentInputRef.current]);
+  }, [isKeyboardOpen, commentInputRef?.current]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of optional chaining in the dependency array of `useEffect` is unnecessary and could lead to unexpected behavior. The `useEffect` hook's dependencies should be stable between renders, and optional chaining can introduce instability if `commentInputRef.current` is undefined at some point. It would be better to just depend on `commentInputRef` itself.

```diff
-  }, [isKeyboardOpen, commentInputRef?.current]);
+  }, [isKeyboardOpen, commentInputRef]);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->